lib/store/index.js

```diff
@@ -174,28 +181,32 @@ export class Store {
     }
   }
 
+  // Returns the embedded grammar for multilanguage, normal grammar otherwise
+  getEmbeddedGrammar(editor: atom$TextEditor): ?atom$Grammar {
+    const grammar = editor.getGrammar();
+    if (!isMultilanguageGrammar(grammar)) {
+      return grammar;
+    }
+
+    const embeddedScope = getEmbeddedScope(
+      editor,
+      editor.getCursorBufferPosition()
+    );
+
+    if (!embeddedScope) return grammar;
+    const scope = embeddedScope.replace(".embedded", "");
+    const grammarForScopeName = atom.grammars.grammarForScopeName(scope);
+    return atom.grammars.grammarForScopeName(scope);
```

Comment:
Is `grammarForScopeName` used?

Reply:
Nope, good catch. We could drop the line above or use the variable name.

Reply:
either is fine